doc/classes/Engine.xml

```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="Engine" inherits="Object" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
+		Access to basic engine properties.
 	</brief_description>
 	<description>
+		The [Engine] class allows you to query and modify basic parameters regarding the run-time characteristics of your game. For example: frames per second, number of frames rendered so far, time scale, etc. 
```

Comment:
It's not necessary to link a class to itself. 

"parameters regarding" is too wordy, and "basic" is too relative. The last part is a sentence fragment. I would just say:

    allows you to query and modify the game's run-time parameters, such as frames per second, time scale, and others.